batch/get-deployed-sha.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -ex
 
-kubectl get --selector=app=batch deployments -o "jsonpath={.items[*].metadata.labels.hail\.is/sha}"
+kubectl get --selector=app=batch-deployment deployments -o "jsonpath={.items[*].metadata.labels.hail\.is/sha}"
```

Comment:
The labels are just the app name:
```
(hail) dking@wmb16-359 # kubectl get --selector=app=hail-ci-deployment deployments -o "jsonpath={.items[*].metadata.labels.hail\.is/sha}"
(hail) dking@wmb16-359 # kubectl get --selector=app=batch-deployment deployments -o "jsonpath={.items[*].metadata.labels.hail\.is/sha}"
(hail) dking@wmb16-359 # kubectl get --selector=app=batch deployments -o "jsonpath={.items[*].metadata.labels.hail\.is/sha}"
dd91f3447b3a
(hail) dking@wmb16-359 # kubectl get --selector=app=hail-ci deployments -o "jsonpath={.items[*].metadata.labels.hail\.is/sha}"
dd91f3447b3a
(hail) dking@wmb16-359 # 
```
```
(hail) dking@wmb16-359 # kubectl describe deployment batch-deployment
Name:                   batch-deployment
Namespace:              default
CreationTimestamp:      Mon, 27 Aug 2018 21:23:33 -0400
Labels:                 app=batch
                        hail.is/sha=dd91f3447b3a
Annotations:            deployment.kubernetes.io/revision=85
                        kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"apps/v1beta2","kind":"Deployment","metadata":{"annotations":{},"labels":{"hail.is/sha":"dd91f3447b3a"},"name":"batch-deployment","namesp...
Selector:               app=batch
Replicas:               1 desired | 1 updated | 1 total | 1 available | 0 unavailable
StrategyType:           RollingUpdate
MinReadySeconds:        0
RollingUpdateStrategy:  25% max unavailable, 25% max surge
Pod Template:
  Labels:  app=batch
```

Reply:
Oops, my bad.  I thought I saw some unnecessary redeploys and thought this explained it.  I'll back out.